From 125b32d35e227b47040cc2e105771bb184c65f87 Mon Sep 17 00:00:00 2001
From: Nicolas Viennot <Nicolas.Viennot@twosigma.com>
Date: Tue, 28 Dec 2021 19:30:09 +0000
Subject: [PATCH 105/120] ci: test criu-image-streamer with all tests

All the bugs that were in the way got fixed. We can enable all tests.

Signed-off-by: Nicolas Viennot <Nicolas.Viennot@twosigma.com>
---
 scripts/ci/run-ci-tests.sh | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/scripts/ci/run-ci-tests.sh b/scripts/ci/run-ci-tests.sh
index 7eab9f2dc..d0cd55f7c 100755
--- a/scripts/ci/run-ci-tests.sh
+++ b/scripts/ci/run-ci-tests.sh
@@ -62,11 +62,9 @@ ci_prep () {
 }
 
 test_stream() {
-	# We must test CRIU features that dump content into an image file to ensure
-	# streaming compatibility.
-	STREAM_TEST_PATTERN='.*(ghost|fifo|unlink|memfd|shmem|socket_queue).*'
+	# Testing CRIU streaming to criu-image-streamer
 	# shellcheck disable=SC2086
-	./test/zdtm.py run --stream -p 2 --keep-going -T "$STREAM_TEST_PATTERN" $ZDTM_OPTS
+	./test/zdtm.py run --stream -p 2 --keep-going -a $ZDTM_OPTS
 }
 
 print_header() {
-- 
2.34.1

